components/integrations/confirm_integration/confirm_integration.jsx

```diff
@@ -51,152 +51,148 @@ export default class ConfirmIntegration extends React.Component {
-                    </p>
-                );
-            }
-        } else if (this.state.type === Constants.Integrations.INCOMING_WEBHOOK) {
-            const incomingHook = this.props.incomingHooks[this.state.id];
-            if (incomingHook) {
-                headerText = (
-                    <FormattedMessage
-                        id={'installed_incoming_webhooks.header'}
-                        defaultMessage='Incoming Webhooks'
+                </p>
+            );
+            tokenText = (
+                <p className='word-break--all'>
+                    <FormattedMarkdownMessage
+                        id='add_command.token'
+                        defaultMessage='**Token**: {token}'
+                        values={{
+                            token: command.token,
+                        }}
                     />
-                );
-                helpText = (
-                    <p>
-                        <FormattedHTMLMessage
-                            id='add_incoming_webhook.doneHelp'
-                            defaultMessage='Your incoming webhook has been set up. Please send data to the following URL (see <a href=\"https://docs.mattermost.com/developer/webhooks-incoming.html\">documentation</a> for further details).'
-                        />
-                    </p>
-                );
-                tokenText = (
-                    <p className='word-break--all'>
-                        <FormattedHTMLMessage
-                            id='add_incoming_webhook.url'
-                            defaultMessage='<b>URL</b>: {url}'
-                            values={{
-                                url: getSiteURL() + '/hooks/' + incomingHook.id,
-                            }}
-                        />
-                    </p>
-                );
-            }
-        } else if (this.state.type === Constants.Integrations.OUTGOING_WEBHOOK) {
-            const outgoingHook = this.props.outgoingHooks[this.state.id];
-            if (outgoingHook) {
-                headerText = (
-                    <FormattedMessage
-                        id={'installed_outgoing_webhooks.header'}
-                        defaultMessage='Outgoing Webhooks'
+                </p>
+            );
+        } else if (this.state.type === Constants.Integrations.INCOMING_WEBHOOK && incomingHook) {
+            headerText = (
+                <FormattedMessage
+                    id={'installed_incoming_webhooks.header'}
+                    defaultMessage='Incoming Webhooks'
+                />
+            );
+            helpText = (
+                <p>
+                    <FormattedMarkdownMessage
+                        id='add_incoming_webhook.doneHelp'
+                        defaultMessage='Your incoming webhook has been set up. Please send data to the following URL (see [documentation](https://docs.mattermost.com/developer/webhooks-incoming.html) for further details).'
```

Comment:
Please have all the links with `!` (open in new tab), like:
`[documentation](!https://docs.mattermost.com/developer/webhooks-incoming.html)`